src/test/java/io/vertx/test/core/FutureTest.java

```diff
@@ -205,6 +206,13 @@ public void testFailurFutureWithNullFailure() {
   }
 
   @Test
+  public void testOrElse() {
+    assertEquals("bye", Future.future().orElse("bye"));
```

Comment:
I think we should drop the `orElse`, anyway the point of this PR was about the `map` that is more useful and works in all cases.